co2calculator/data_handlers.py

```diff
@@ -0,0 +1,71 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+"""Reader class to retrieve co2 factors from database"""
+import warnings
+from pathlib import Path
+import pandas as pd
+from .exceptions import ConversionFactorNotFound
+
+script_path = str(Path(__file__).parent)
+
+
+class EmissionFactors:
+    def __init__(self):
+        """Init"""
+        self.emission_factors = pd.read_csv(
+            f"{script_path}/../data/emission_factors.csv"
+        )
+        self.column_names = self.emission_factors.columns
+
+    def get(self, parameters: dict):
+        """
+        Returns factors from the database
+        :param parameters:
+        :type parameters:
+        :return:
+        :rtype:
+        """
+        selected_factors = self.emission_factors
+
+        for k, v in parameters.items():
+            # shortterm hack to make it work until co2 factors are updated
```

Comment:
I think we should add some keywoard like `TODO` in these comments so we can finde them vai text search.

Reply:
added